tests/schemas/entity.py

```diff
@@ -70,6 +70,22 @@ def test_filter_entities_regex(self):
         self.assertEqual(total, 1)
         self.assertEqual(entities[0], self.vuln1)
 
+    def test_filter_entities_time(self):
+        entities, total = Entity.filter({"created": "2020-01-01"})
```

Comment:
Looking at this test, does it means from UI you have to type `created=2020-01-01` and so to use gt or lt you need to type `created=>2020-01-01` or `created=<2020-01-01`

Reply:
That's correct. Happy to revisit it if you think it's not intuitive enough. doing `created>2020-01-01` implies some more complex changes to the way we parse the query in the UI. Not impossible to do, but much more complex.